pysat/tests/test_instrument.py

```diff
@@ -212,6 +372,25 @@ def test_download_updated_files(self, caplog):
         # Update local file list
         assert "Updating pysat file list" in caplog.text
 
+    def test_download_updated_files_file_bounds_default(self, caplog):
+        """Ensure that Instrument bound are updated, post download, when
+        bounds are set as first and last file."""
+        # set bounds to first and last file
+        self.testInst.bounds = (self.testInst.files[0], self.testInst.files[-1])
+        # now download files
+        with caplog.at_level(logging.INFO, logger='pysat'):
+            self.testInst.download_updated_files()
+        # Perform a local search
+        assert "files locally" in caplog.text
+        # New files are found
+        assert "that are new or updated" in caplog.text
+        # download new files
+        assert "Downloading data to" in caplog.text
+        # Update local file list
+        assert "Updating pysat file list" in caplog.text
+        # default bounds update
```

Comment:
All of this is already in `test_download_updated_files`.  Recommend combining using parametrize.